src/GrampsJs.js

```diff
@@ -582,6 +578,13 @@ export class GrampsJs extends LitElement {
     window.addEventListener('db:changed', () => this._loadDbInfo(false))
     this.addEventListener('drawer:toggle', this._toggleDrawer)
     window.addEventListener('keydown', (event) => this._handleKey(event))
+
+    const browserLang = this._getBrowserLanguage()
+    if (browserLang) {
+      updateSettings({ lang: browserLang })
+      this.settings = getSettings()
+      this._loadStrings(grampsStrings, this.settings.lang)
```

Comment:
You cannot do this in connectedCallback as the user might not be logged in or even registered, so the call will fail.

The original call you removed was done as part of the render method at a point where it's guaranteed we are already authenticated.

Reply:
On second thought (see discussion below), if we rewrite `loadStrings` to fall back to frontend strings only in case of no authorization we can do it, but we have to make sure backend strings are loaded once we are authorized.